pkg/engine/validation.go

```diff
@@ -132,8 +134,21 @@ func validateResource(ctx context.Context, log logr.Logger, rclient registryclie
 				if !matches(log, rule, enginectx) {
 					return nil
 				}
+				// if matches, check if there is corresponding policy exception
+				isExcepted, exceptionName := matchesException(log, enginectx, rule, peLister)
 				log.V(3).Info("processing validation rule", "matchCount", matchCount, "applyRules", applyRules)
 				enginectx.jsonContext.Reset()
+				if isExcepted {
+					// log rule was skipped because of an exception
+					// create a skip response
+					// increase metrics
+					ruleResp := &response.RuleResponse{
+						Name: rule.Name,
+						Message: "Rule skipped because of PolicyException" + exceptionName,
```

Comment:
can we add the namespace of the policy exception resource?

Reply:
exceptionName and policyName should always be a name in the form of `<namespace>/<name>`.
We need this to support both clustered and namespace policies in exceptions anyway.

Reply:
Have changed it to `exceptionInfo`, which is in the form of `<exceptionNamespace>/<exceptionName>`.